raiden-dapp/src/services/raiden-service.ts

```diff
@@ -52,7 +52,10 @@ export default class RaidenService {
 
   async ensResolve(name: string): Promise<string> {
     try {
-      return await this.raiden.resolveName(name);
+      const address = await this.raiden.resolveName(name);
+      const { available } = await this.raiden.getAvailability(address);
+      this.store.commit('updatePresence', { [address]: available });
```

Comment:
Is there any specific reason why you update the presense here instead of the `getAvailability` function

Reply:
Nice catch! Actually it's not necessary here to update the state. I will get rid of it and test it quickly if everything still works. The state positive (read: `available=true`) updates should all be handled here: https://github.com/raiden-network/light-client/pull/653/files#diff-ddfc85a29ab30ce1822e4b860e4305d2R121